deploy/kubectl_helpers/utils/get_pod_info.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
```

Comment:
I would probably `set -u` here too. The `get` later on isn't doing anything destructive, but I think it'll help a bit with debugging and maintainability.